tsconfig.json

```diff
@@ -13,10 +13,15 @@
       "es2016",
       "es2017"
     ],
+    "typeRoots": [
+      "node_modules/@types",
```

Comment:
Why did you need this?

Reply:
I thought defining explicitly the typeRoots would require to redefine the existing ones.